app/src/main/java/com/brandon/forzenbook/view/composables/ErrorComposables.kt

```diff
@@ -77,9 +77,9 @@ fun FakeCreateAccountScreen() {
             onTextChange = { },
         )
         InputInfoTextField(
-            hint = resources.getString(R.string.createAccountPasswordHint),
+            hint = resources.getString(R.string.createAccountcodeHint),
             onTextChange = { },
-            characterLimit = ForzenTopLevelViewModel.PASSWORD_CHAR_LIMIT,
+            characterLimit = ForzenTopLevelViewModel.code_CHAR_LIMIT,
```

Comment:
Constants should have ALL their letters capitalized, please change

Reply:
Fixed this